config/initializers/test_track_load_timestamp.rb

```diff
@@ -0,0 +1 @@
+TestTrack.load_build_timestamp
```

Comment:
do we need `test_track` in the filename? may just `load_build_timestamp`? since test_track is kinda implied